frontend-react/src/App.tsx

```diff
@@ -64,6 +64,8 @@ const App = () => {
         navigate(toRelativeUrl(originalUri, window.location.origin));
     };
 
+    initializeSessionBroadcastChannel(OKTA_AUTH); // for cross-tab login/logout
```

Comment:
Let's move this call outside of the component body.  It's not a big deal because we're early terminating if we've already initialized, but there's no reason to call this on each re-render of the component:

```js
initializeSessionBroadcastChannel(OKTA_AUTH);

const App = () => { ... };
```

Reply:
Agreed good call 👍